packages/core/parcel-bundler/src/SourceMap.js

```diff
@@ -57,6 +63,7 @@ class SourceMap {
       }
     } else {
       if (!map.eachMapping) {
+        // not a SourceMapConsumer
```

Comment:
This check is for a SourceMap instance that has been converted to JSON (to transfer it from worker=>Master).
So this comment is slightly incorrect?